packages/chain-helpers/package.json

```
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "@kiltprotocol/config": "workspace:*",
+    "@kiltprotocol/type-definitions": "0.1.3",
```

```suggestion
    "@kiltprotocol/type-definitions": "^0.1.3",
```

Would stop us from needing to creating PRs to update it

---

oh that's intended, type definitions break to easily with additional work required on the sdk. We'd have to be really careful with semversioning if we're not pinning